src/Meshes/geometry.ts

```diff
@@ -239,7 +249,7 @@ export class Geometry implements IGetSetVerticesData {
      * @param kind defines the data kind (Position, normal, etc...)
      * @param data defines the vertex data to use
      * @param updatable defines if the vertex must be flagged as updatable (false as default)
-     * @param stride defines the stride to use (0 by default). This value is deduced from the kind value if not specified
+     * @param stride defines the stride size to use in the position buffer i.e. the distance between two positions in the positions array. (Default is 3)

```

Comment:
This is a breaking change..It should remain autodetect

Reply:
You're right and the new comment is wrong, copy/paste is evil...

Reply:
I did not change anything in this function, so I'll let the jsDoc unchanged

Reply:
yep no worries :D

Reply:
change reverted. And happy new year folks :)

Reply:
Happy new year!!!